.vscode/settings.json

```diff
@@ -0,0 +1,2 @@
+{
```

Comment:
why committing this?

Reply:
Apologize, that was a mistake. That should not be there